src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,115 @@
+                 :hole-radius   {:circular 9 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     18
+                 :hole-x        11
+                 :hole-y        -1}})
+
+;; TODO - Add avatar components for other types once implemented
+(defn avatar [item type size]
+  (case type
+    :user [user-avatar/user-avatar
+           (merge {:ring?             false
+                   :status-indicator? false
+                   :size              (case size 32 :small 24 :xs 16 :xxxs)}
+                  item)]))
+
+(defn list-item [idx type size item list-size margin-left
+                 hole-size hole-radius hole-x hole-y]
+  (let [last-item? (= idx (- list-size 1))]
+    [rn/hole-view {:style {:margin-left   (if (= idx 0) 0 margin-left)}
+                   :holes (if last-item? []
+                              [{:x             hole-x
+                                :y             hole-y
+                                :width         hole-size
+                                :height        hole-size
+                                :borderRadius  hole-radius}])}
+     [avatar item type size]]))
+
+(defn get-overflow-color [transparent? transparent-color light-color dark-color]
+  (if transparent?
+    transparent-color
+    (colors/theme-colors light-color dark-color)))
+
+(defn overflow-label [label size transparent? border-radius margin-left]
+  [rn/view {:style {:width            size
+                    :height           size
+                    :margin-left      margin-left
+                    :border-radius    border-radius
+                    :justify-content  :center
+                    :align-items      :center
+                    :background-color (get-overflow-color
+                                       transparent?
+                                       colors/white-opa-10
+                                       colors/neutral-20
+                                       colors/neutral-70)}}
+   (if (= size 16)
+     [quo2.icons/icon :main-icons2/more {:size 12
+                                         :color (get-overflow-color
+                                                 transparent?
+                                                 colors/white-opa-70
+                                                 colors/neutral-50
+                                                 colors/neutral-40)}]
+     [quo2.text/text {:size   (if (= size 32) :paragraph-2 :label)
+                      :weight :medium
+                      :style  {:color       (get-overflow-color
+                                             transparent?
+                                             colors/white-opa-70
+                                             colors/neutral-60
+                                             colors/neutral-40)
+                               :margin-left -2}}
+      ;; If overflow label is below 100, show label as +label (ex. +30), else just show 99+
```

Comment:
is this comment really needed?

Reply:
https://github.com/status-im/status-mobile/pull/13830#discussion_r967197344